packages/x-date-pickers/src/internals/hooks/usePicker/usePicker.ts

```diff
@@ -90,7 +82,10 @@ export const usePicker = <
     shouldRestoreFocus: pickerViewsResponse.shouldRestoreFocus,
 
     // Picker layout
-    layoutProps: pickerLayoutResponse.layoutProps,
+    layoutProps: {
```

Comment:
I will do follow up to remove those props as well :laughing: 
Most of them are forwarded directly to the slots handled by the layout so I will probably edit those slots first.